dataservice/api/read_group/schemas.py

```diff
@@ -0,0 +1,43 @@
+from marshmallow_sqlalchemy import field_for
+
+from dataservice.api.common.schemas import BaseSchema
+from dataservice.api.read_group.models import ReadGroup
+from dataservice.api.common.validation import (validate_positive_number,
+                                               enum_validation_generator)
+from dataservice.extensions import ma
+
+PAIRED_END_ENUM = {1, 2}
+QUALITY_SCALE_ENUM = {'Illumina13', 'Illumina15', 'Illumina18',
+                      'Solexa', 'Sanger'}
+
+
+class ReadGroupSchema(BaseSchema):
+
+    paired_end = field_for(ReadGroup, 'paired_end',
+                           validate=enum_validation_generator(
+                               PAIRED_END_ENUM))
+
+    lane_number = field_for(ReadGroup, 'lane_number',
+                            validate=validate_positive_number)
+
+    quality_scale = field_for(ReadGroup, 'quality_scale',
+                              validate=enum_validation_generator(
+                                  QUALITY_SCALE_ENUM, common=False))
+
+    genomic_file_id = field_for(ReadGroup, 'genomic_file_id',
+                                required=True, load_only=True)
+
+    class Meta(BaseSchema.Meta):
+        resource_url = 'api.read_groups'
+        collection_url = 'api.read_groups_list'
+        model = ReadGroup
+        exclude = (BaseSchema.Meta.exclude +
+                   ('read_groups', ) +
```

Comment:
Remove the `'read_groups'` attribute since its not actually an attribute on the ReadGroup model. Change `'genomic_files' ` to `'genomic-file'` since a read group has 1 genomic_file as its parent.